File: 0485-max-consecutive-ones/0485-max-consecutive-ones.c
int findMaxConsecutiveOnes(int* nums, int numsSize) {
    int i=0;
    int count = 0;
    int max = 0;
    while(i!=numsSize){
        if(nums[i]!=1){
            if(max<count){
                max = count;
            }
            count = 0;
        }
        else{
            count += 1;
        }
        i += 1;
    }
    if(max<count){
        max = count;
    }
    return max;
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if 0 not in nums:
            return len(nums)
        elif 1 not in nums:
            return 0
        maxi = 0
        countinous = 0
        for i in range(len(nums)):
            if nums[i]!=1:
                maxi = max(maxi, countinous)
                countinous=0
            else:
                countinous += 1
        maxi = max(maxi, countinous)
        return maxi        